### DISCUSSION TITLE
[Spritepack issue]How to make thumbnails show up?

### DISCUSSION THREAD

**Discussion by MoriMoriMorii** [Monika-After-Story/MonikaModDev#8334] [QA - code and content creation (technical)] [answered]
For context, I'm making a spritepack consisting of 3 things, and only one of them has a working thumbnail(The necklace), I'm not sure what I'm doing wrong, I've reached out on Reddit but no one could answer my question, so I thought here would be a good place to do so.
Attached are the 3 JSON files(Text files for git) I've been working with.
Thanks!
[Dr_memento_mori_acs_ribbon_black-and-white.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7643760/Dr_memento_mori_acs_ribbon_black-and-white.txt)
[Dr_memento_mori_clothes_ua_sweater.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7643761/Dr_memento_mori_clothes_ua_sweater.txt)
[Dr_memento_mori_acs_coal_necklace_short.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7643762/Dr_memento_mori_acs_coal_necklace_short.txt)

**Comment by ThePotatoGuy** [comment, marked as answer]
the thumbnail image in the `thumbs/` [folder](https://github.com/Monika-After-Story/MonikaModDev/tree/master/Monika%20After%20Story/game/mod_assets/thumbs) needs to match what you put in the `"thumb"` property in the JSON, except for the acs/clothes/hair prefix.

**Reply by MoriMoriMorii** [reply]
So do I remove the prefix on either the image or document? All the filenames seem to be correct

**Reply by MoriMoriMorii** [reply]
I had to remove the prefix and change a few things, thanks!
